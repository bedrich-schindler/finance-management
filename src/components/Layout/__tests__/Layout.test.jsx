import React from 'react';
import { shallow } from 'enzyme';
import { Layout } from '..';

describe('rendering', () => {
  it('renders correctly', () => {
    const tree = shallow(
      <Layout>
        <p>Content</p>
      </Layout>,
    );

    expect(tree).toMatchSnapshot();
  });
});
