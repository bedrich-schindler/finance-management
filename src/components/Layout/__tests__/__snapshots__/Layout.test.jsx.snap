// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering renders correctly 1`] = `
<div>
  <div
    className="root"
  >
    <WithStyles(Header)
      isMenuOpened={false}
      onOpenMenu={[Function]}
      title="Finance management"
    />
    <WithStyles(withRouter(Menu))
      isMenuOpened={false}
      onCloseMenu={[Function]}
    />
  </div>
  <main
    className="content"
  >
    <p>
      Content
    </p>
  </main>
</div>
`;
